script/docker_dev_server

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+bundle install --local --quiet --jobs 4 || bundle check
+
+if [ ! -f .env ]; then
+    cp .env.bootstrap .env
+    ./bin/decode_dot_env .env
+fi
+
+if [ -f db/development.sqlite ]; then
+    bundle exec rake db:migrate
+else
+    bundle exec rake db:setup
+fi
+
+exec bundle exec puma -C config/puma.rb
```

Comment:
docker things should not be part of this PR

Reply:
Will remove after final review, otherwise I can't get the code to run...